### PR TITLE
Updated dictionary for #2411

### LINKED ISSUE #2411: Codespell doesn't identify issues in docstrings and markdown
Hi there,

So I am running codespell using pre-commit and it doesn't identify errors inside docstrings and markdown files. See for example this commit in Starlite --> https://github.com/starlite-api/starlite/pull/283/commits/26afa44cb8e044eda0f9440da5d5cc9d37e2ef09, I fixed typos in it using intellij code inspection, all of which were not identified by codespell at all.

### PR DESCRIPTION
**Pull request by matlupi**
Adds missing typos in #2411.
Close #2411.

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -8115,6 +8116,7 @@ bootstraper->bootstrapper
 bootstrapers->bootstrappers
 bootstraping->bootstrapping
 bootstras->bootstraps
+bootup->boot-up
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:8119]: May I suggest end-users can choose between `boot up` and `boot-up`?
```suggestion
bootup->boot up, boot-up,
```

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -23222,6 +23229,7 @@ exceprts->excerpts
 exceptation->expectation
 exceptin->excepting, exception, expecting, accepting,
 exceptins->exceptions, excepting,
+exceptioin->exception
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:23232]: I suggest adding plural:
```suggestion
exceptioin->exception
exceptioins->exceptions
```

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -28898,6 +28908,7 @@ implemetning->implementing
 implemetns->implements
 implemets->implements
 implemnt->implement
+implemnt->implement
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:28911]: Isn't the above a duplicate? No worries, that's the perfect oocasion to add all verbal forms:
```suggestion
implemntation->implementation
implemnted->implemented
implemnting->implementing
implemnts->implements
```

File: codespell_lib/data/dictionary.txt [hunk 4]
```diff
@@ -44657,6 +44670,10 @@ redandant->redundant
 redction->reduction, redaction,
 redeable->readable
 redeclaation->redeclaration
+redeclare->re-declare
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:44673]: * [OED](https://www.oed.com/dictionary/redeclare_v) shows `redeclare` is a valid word.
* [The Blue Book of Grammar and Punctuation](https://www.grammarbook.com/blog/hyphens/hyphens-with-the-prefix-re/), among other sources, recommends to **avoid the hyphen** in most cases.
* The[ Google Ngram Viewer](https://books.google.com/ngrams/graph?content=redeclare%2Cre-declare&year_start=1800&year_end=2019&corpus=en-2019) shows that `redeclare` is actually more common.

Comment (matlupi) [reply codespell_lib/data/dictionary.txt:44673]:
```suggestion
```

File: codespell_lib/data/dictionary.txt [hunk 5]
```diff
@@ -44657,6 +44670,10 @@ redandant->redundant
...
+redeclared->re-declared
+redeclares->re-declares
+redeclaring->re-declaring
```
Comment (matlupi) [inline codespell_lib/data/dictionary.txt:44676]:
```suggestion
```

File: codespell_lib/data/dictionary.txt [hunk 6]
```diff
@@ -44657,6 +44670,10 @@ redandant->redundant
 redction->reduction, redaction,
 redeable->readable
 redeclaation->redeclaration
+redeclare->re-declare
+redeclared->re-declared
+redeclares->re-declares
```
Comment (matlupi) [inline codespell_lib/data/dictionary.txt:44675]:
```suggestion
```

File: codespell_lib/data/dictionary.txt [hunk 7]
```diff
@@ -44657,6 +44670,10 @@ redandant->redundant
 redction->reduction, redaction,
 redeable->readable
 redeclaation->redeclaration
+redeclare->re-declare
+redeclared->re-declared
```
Comment (matlupi) [inline codespell_lib/data/dictionary.txt:44674]:
```suggestion
```

File: codespell_lib/data/dictionary.txt (hunk 4 above)
Comment (matlupi) [reply codespell_lib/data/dictionary.txt]: I did implement the opposite following the message above.
Now re-declare maps to redeclare (same goes for all verbal forms).

File: codespell_lib/data/dictionary.txt [hunk 8]
```diff
@@ -13018,6 +13020,7 @@ concered->concerned
 concerened->concerned
 concering->concerning
 concerntrating->concentrating
+concerte->concrete
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:13023]: - `e` is near `s` on my keyboard. Perhaps add `concerts`, as well as `concert` and even `concerto`?
- Also add `concerted`?

File: codespell_lib/data/dictionary.txt [hunk 9]
```diff
@@ -22053,6 +22056,10 @@ enforceing->enforcing
 enforcmement->enforcement
 enforcment->enforcement
 enfore->enforce
+enforece->enforce
+enforeced->enforced
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:22060]: Add noun?
```
enforecer->enforcer
enforecers->enforcers
```

File: codespell_lib/data/dictionary.txt [hunk 10]
```diff
@@ -38660,6 +38671,8 @@ othopaedic->orthopaedic
 othopaedics->orthopaedics
 othopedic->orthopedic
 othopedics->orthopedics
+othre->other
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:38674]: Perhaps add `ochre`?

**Review by DimitriPapadopoulos — APPROVED** [review]

**Comment by DimitriPapadopoulos** [comment]
@matlupi Thank you for your contributions.

**Pull request merged**